addons/mail/static/tests/messaging_menu/messaging_menu.test.js

```diff
@@ -1016,6 +1016,47 @@ test("Attachment-only message preview shows file names (3 files)", async () => {
     });
 });
 
+test("render message preview with message seen indicator in messaging menu", async () => {
+    const pyEnv = await startServer();
+    const [Thomas, Arthur] = pyEnv["res.partner"].create([
+        { name: "Thomas Shelby" },
+        { name: "Arthur Shelby" },
+    ]);
+    const channelId = pyEnv["discuss.channel"].create({
+        name: "Small Heath",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ partner_id: Thomas }),
+            Command.create({ partner_id: Arthur }),
```

Comment:
No capital letter on variables unless they are classes, otherwise the code highlighting is misleading. _(referring to `[Thomas, Arthur]`)_